okhttp/src/main/java/okhttp3/internal/concurrent/TaskRunner.kt

```diff
@@ -126,13 +127,13 @@ class TaskRunner(
     fun coordinatorWait(taskRunner: TaskRunner, nanos: Long)
   }
 
-  class RealBackend : Backend {
+  internal class RealBackend : Backend {
     private val coordinatorExecutor = ThreadPoolExecutor(
         0, // corePoolSize.
         1, // maximumPoolSize.
         60L, TimeUnit.SECONDS, // keepAliveTime.
-        SynchronousQueue(),
-        threadFactory("OkHttp Task Coordinator", false)
+        LinkedBlockingQueue<Runnable>(),
```

Comment:
I had to switch this to prevent RejectedExecutionExceptions when a coordinator quickly exits and then starts again.